tests/conftest.py

```diff
@@ -450,3 +453,21 @@ def _reset_log():
     with other tests that depend on it.
     """
     log.reset()
+
+
+@pytest.fixture
+def make_config_file(tmpdir_cwd):
```

Comment:
Definitely add a docsting — it's shown in the `pytest --fixtures` output and is a part of good DX/contributore experience.

Reply:
Marking this as resolved.